### DISCUSSION TITLE
How to add a project post on the blog

### DISCUSSION THREAD

**Discussion by Vrinda73** [bitcraze/discussions#902] [Q&A]
Hello!
I recently got successful in implementing a project involving Crazyflie 2.1. A big thanks to the Bitcraze community. I wanted to share what I did with the community on the blog. Please can you help me get started. 
Thank you!

**Comment by krichardsson** [comment]
Sounds exciting!
Just drop us an email on contact@bitcraze.io and we will help you out

**Reply by Vrinda73** [reply]
Thank you very much Kristoffer!
